src/applications/pre-need/config/form.jsx

```diff
@@ -7,15 +7,15 @@ import environment from '../../../platform/utilities/environment';
 import preSubmitInfo from '../../../platform/forms/preSubmitInfo';
 
 import * as address from '../definitions/address';
-import currentOrPastDateUI from 'us-forms-system/lib/js/definitions/currentOrPastDate';
-import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
-import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
```

Comment:
LMK how you feel about this structure- `forms` is taken so i figured `forms-system` would work. i verified this form works after doing this import.

Reply:
I'm going to move this to another PR so i can just gather feedback there.